automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/Leonardo.scala

```diff
@@ -27,51 +27,51 @@ object Leonardo extends RestClient with LazyLogging {
   }
 
   object cluster {
-    def handleClusterResponse(response: String): Cluster = {
+    def handleClusterResponse(response: String): ClusterCopy = {
       val res = for {
         json <- io.circe.parser.parse(response)
-        r <- json.as[Cluster]
+        r <- json.as[ClusterCopy]
       } yield r
 
       res.getOrElse(throw new Exception("Failed to parse list of clusters response"))
     }
 
-    def handleClusterSeqResponse(response: String): List[Cluster] = {
+    def handleClusterSeqResponse(response: String): List[ClusterCopy] = {
       val res = for {
         json <- io.circe.parser.parse(response)
-        r <- json.as[List[Cluster]]
+        r <- json.as[List[ClusterCopy]]
       } yield r
 
       res.getOrElse(throw new Exception("Failed to parse list of clusters response"))
     }
 
     def clusterPath(googleProject: GoogleProject,
-                    clusterName: ClusterName,
+                    clusterName: ClusterNameCopy,
                     version: Option[ApiVersion] = None): String = {
       val versionPath = version.map(_.toUrlSegment).getOrElse("")
       s"api/cluster${versionPath}/${googleProject.value}/${clusterName.string}"
     }
 
-    def list(googleProject: GoogleProject)(implicit token: AuthToken): Seq[Cluster] = {
+    def list(googleProject: GoogleProject)(implicit token: AuthToken): Seq[ClusterCopy] = {
       logger.info(s"Listing active clusters in project: GET /api/clusters/${googleProject.value}")
       handleClusterSeqResponse(parseResponse(getRequest(url + "api/clusters")))
     }
 
-    def listIncludingDeleted(googleProject: GoogleProject)(implicit token: AuthToken): Seq[Cluster] = {
+    def listIncludingDeleted(googleProject: GoogleProject)(implicit token: AuthToken): Seq[ClusterCopy] = {
       val path = s"api/clusters/${googleProject.value}?includeDeleted=true"
       logger.info(s"Listing clusters including deleted in project: GET /$path")
       handleClusterSeqResponse(parseResponse(getRequest(s"$url/$path")))
     }
 
-    def create(googleProject: GoogleProject, clusterName: ClusterName, clusterRequest: ClusterRequest)(
```

Comment:
`ClusterName` is primitive enough to be shared?

Reply:
Yeah.. can remove `ClusterNameCopy`